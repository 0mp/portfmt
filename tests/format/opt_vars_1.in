OPENMPI2_VARS=		cc=${LOCALBASE}/mpi/openmpi2/bin/mpicc \
			cxx=${LOCALBASE}/mpi/openmpi2/bin/mpicxx
