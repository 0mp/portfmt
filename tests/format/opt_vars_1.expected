OPENMPI2_VARS=	CC=${LOCALBASE}/mpi/openmpi2/bin/mpicc \
		CXX=${LOCALBASE}/mpi/openmpi2/bin/mpicxx
